ExampleCodes/FFT/EnergySpectrum/main.cpp

```diff
@@ -55,6 +57,12 @@ int main (int argc, char* argv[])
                 Real value = amrex::norm(cxa[b](i,j,k))
                     +        amrex::norm(cya[b](i,j,k))
                     +        amrex::norm(cza[b](i,j,k));
+		// Account for Hermitian symmetry in x-direction
+	        // Hermitian symmetry Y[nx-i,j,k] = Y[i,j,k]*
+                if ((i > 0) && (2*i != nx)) {
+		    // Multiply by 2 because we have +ki and -ki
+                    value *= Real(2.0);
+                }
```

Comment:
```suggestion
                // Account for Hermitian symmetry in x-direction
                // Hermitian symmetry Y[nx-i,j,k] = Y[i,j,k]*
                if ((i > 0) && (2*i != nx)) {
                    // Multiply by 2 because we have +ki and -ki
                    value *= Real(2.0);
                }
```